docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+web:
+    build: .
+    volumes:
+        - .:/app
```

Comment:
to improve performances on MacOS and Windows (and fix permission issues), you can define volumes for 
```
 volumes:
    - .:/app
    - /app/app/cache
    - /app/app/logs
```